eth/.solhintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 contracts/Migrations.sol
 contracts/WETH9.sol
+contracts/Token.sol
```

Comment:
Suggestion: I would throw in the entire `gnosis-1.3.0` here - external code that we trust, we won't be working on it in any way

Reply:
Or is it already omitted? Anyway, wouldn't format it